### ISSUE TITLE
Add approximate word counts to text fields

### ISSUE THREAD

**Issue by gabestein** [pubpub/pubpub#363]
As a community manager, I want text fields with character counts to include approximate word counts, so I can better communicate with authors who are writing for those fields.

**Requested by**
reagle on discourse
https://discourse.knowledgefutures.org/t/280-character-limit/134/4

**Comment by reagle** [comment]
To clarify, I also wanted more characters. Absent that, the "tweet" length and a real abstract use on the front page of a PubPub could be two different fields.

**Comment by gabestein** [comment]
Thanks. As I said on Discourse, we're planning to add an abstract field in future releases, and we will consider making that field available for descriptions on pages. 

The update we can commit to now, which this issue addresses, is adding approximate word counts to existing fields to make it easier for authors to reason.

**Comment by reagle** [comment]
If it must be a strict character count, I'd keep it at that. It was frustrating to be told 50 words and find out it was 280 characters.

**Comment by gabestein** [comment]
As I wrote in the Discourse thread, the idea is to give both the strict character count and a bit of a signpost to help authors who think more in words understand what 280 characters mean. Something like: `280 characters (about 35 words)`.

I actually did a fair amount of testing on these types of directives and writer behavior at a previous job that relied heavily on the effectiveness of social media previews, and found this approach was quite effective.
